docs/reference/indices/field-usage-stats.asciidoc

```diff
@@ -0,0 +1,143 @@
+=== Field usage stats API
+++++
+<titleabbrev>Field usage stats</titleabbrev>
+++++
+
+experimental[]
+
+Returns field usage information for each field of an index.
+Field usage statistics are automatically captured when
+queries are runnning on a cluster. A shard-level search
+request that accesses a given field, even if multiple times
+during that request, is counted as a single use.
+
+[source,console]
+--------------------------------------------------
+GET /my-index-000001/_field_usage_stats
+--------------------------------------------------
+// TEST[setup:messages]
+
+[[field-usage-stats-api-request]]
+==== {api-request-title}
+
+`GET /<index>/_field_usage_stats
+
+[[field-usage-stats-api-request-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage`
+<<privileges-list-indices,index privilege>> for the target index or index alias.
+
+[[field-usage-stats-api-path-params]]
+==== {api-path-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index]
+
+[[field-usage-stats-api-query-params]]
+==== {api-query-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=wait_for_active_shards]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+`fields`::
++
+--
+(Optional, string)
+Comma-separated list or wildcard expressions of fields
+to include in the statistics.
+--
+
+[[field-usage-stats-api-example]]
+==== {api-examples-title}
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+POST /my-index-000001/_search
+{
+  "query" : {
+    "match" : { "context" : "bar" }
+  },
+  "aggs": {
+    "message_stats": {
+      "string_stats": {
+        "field": "message.keyword",
+        "show_distribution": true  <1>
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TEST[setup:messages]
+
+//////////////////////////
+
+The following request retrieves field usage information of index `my-index-000001`.
+
+[source,console]
+--------------------------------------------------
+GET /my-index-000001/_field_usage_stats
+--------------------------------------------------
+// TEST[continued]
+
+The API returns the following response:
+
+[source,console-response]
+--------------------------------------------------
+{
+    "_shards": {
+        "total": 1,
+        "successful": 1,
+        "failed": 0
+    },
+    "my-index-000001": {
+        "all_fields": {
+            "total": "6", <1>
+            "terms" : 1,
+            "proximity" : 1,
+            "stored_fields" : 2,
+            "doc_values" : 1,
+            "points" : 0,
+            "norms" : 1,
+            "term_vectors" : 0,
+            "frequencies" : 1,
```

Comment:
nit: Lucene has two sorts of frequencies: term frequencies and doc frequencies, it might help clarify to rename this `term_frequencies`